test/org/sagebionetworks/bridge/services/FPHSServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package org.sagebionetworks.bridge.services;
+
+import static org.sagebionetworks.bridge.TestConstants.TEST_STUDY;
+import static org.sagebionetworks.bridge.dao.ParticipantOption.EXTERNAL_IDENTIFIER;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.sagebionetworks.bridge.dao.FPHSExternalIdentifierDao;
+import org.sagebionetworks.bridge.dao.ParticipantOption;
+import org.sagebionetworks.bridge.dynamodb.DynamoFPHSExternalIdentifier;
+import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
+import org.sagebionetworks.bridge.exceptions.InvalidEntityException;
+import org.sagebionetworks.bridge.models.accounts.ExternalIdentifier;
+import org.sagebionetworks.bridge.models.accounts.FPHSExternalIdentifier;
+
+import com.google.common.collect.Lists;
+
+public class FPHSServiceTest {
+
+    private FPHSService service;
+    private FPHSExternalIdentifierDao dao;
+    private ParticipantOptionsService optionsService;
+    private ExternalIdentifier externalId;
+    
+    @Before
+    public void before() {
+        externalId =  new ExternalIdentifier("gar");
+        service = new FPHSService();
+        dao = mock(FPHSExternalIdentifierDao.class);
+        optionsService = mock(ParticipantOptionsService.class);
+        
+        service.setFPHSExternalIdentifierDao(dao); 
+        service.setParticipantOptionsService(optionsService);
+    }
+    
+    @Test(expected = InvalidEntityException.class)
+    public void validateIdThrowsException() throws Exception {
+        service.verifyExternalIdentifier(new ExternalIdentifier(""));
+    }
+    
+    @Test
+    public void verifyExternalIdentifierSucceeds() throws Exception {
+        service.verifyExternalIdentifier(externalId);
+        verify(dao).verifyExternalId(externalId);
+    }
```

Comment:
Is it worth adding a unit test for verify() fails? (An actual verify failure, not an empty string check.)